### ISSUE TITLE
Succession() act incorrectly when display Mobjs successively

### ISSUE THREAD

**Issue by cckk4467** [ManimCommunity/manim#1256]
## Description of bug / unexpected behavior

when more than once Write() or FadeIn() sth within Succession(), it will show the mobjs to be displayed before the animation begin
look at the blue triangle at the very beginning of the video below. It's not supposed to be presented before the white line

## Expected behavior
Just to display the mobjs one by one normally

## How to reproduce the issue

<details><summary>Code for reproducing the problem</summary>

```python
class Test(Scene):
    def construct(self):
        tri=Line(UP)
        a=Succession(
            FadeIn(Line()),
            FadeIn(tri),
        )
        self.play(a)
```

</details>

## Additional media files

<details><summary>Images/GIFs</summary>

https://user-images.githubusercontent.com/26384746/113535642-947b2280-9606-11eb-9009-2e26006c8507.mp4

</details>

**Comment by AliNear** [comment]
Yes I have a similar issue with Succession, it displays all the mobjects at fist then runs the animations successively.

**Comment by cobordism** [comment]
There is a more general problem with the way AnimationGroup works (Succession inherits from there).

To fix this properly we need to refactor all of that.

The cause of this issue is that an AnimationGroup is not a group of animations, but an animation on a Group mobject. All mobjects are added to the group, and the initialisation call at Animation.begin() happens before the first animation starts.

**Comment by Kiran-Raj-Dev** [comment]
Fixed via #2396 by the introduction of the `introducer` keyword argument.
